### ISSUE TITLE
Status and Domain are made sequentially, but have no dependency

### ISSUE THREAD

**Issue by kurtisassad** [hicommonwealth/commonwealth#4910]
## Describe the bug
In useInitApp, we call both /domain and /status. These routes do not depend on each other, so they should be made in parallel with Promise.all instead of being run in series.

## To Reproduce
Just navigate to any page, in the network tab you will see:
<img width="1249" alt="Screenshot 2023-08-29 at 8 15 30 AM" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/2de1269a-3976-48fd-bde9-139bec9c9598">

Notice in the waterfall section, these two call timings do not overlap. In the code, they are performed sequentially.

## Expected behavior
These 2 calls happen in parallel (The waterfall bars are allowed to overlap)

## Additional context
In my run above for example. it should reduce the LCP by ~300ms as we don't need to wait on domain before we make the call to status

**Comment by dillchen** [comment]
This is super, super great! @kurtisassad

Semi related, but out of scope. Wondering if status route itself could be parallelized. We've long discussed it but, can we split up the status route to "user settings" and "site settings".

It seems like doing that would be a good start to bringing the round trip time on status down

**Comment by kurtisassad** [comment]
@dillchen I have looked into it, we are already doing this on the status route (All db calls are inside a Promise.all).

Breaking out the status call into multiple calls was more so about lazy loading vs eager loading. Right now we load everything in status eagerly, but we might not need it at that moment on the webpage, leading to the large request time.

Breaking it up, we can load the individual parts lazily when we need them. This would not effect the overall load time, but instead would spread out the load time of status over the webpages that require them (leading to a perceived speedup).

**Comment by jnaviask** [comment]
@kurtisassad @dillchen IMO the main problem with status is the eager loading of all community-related information. In theory we should only load the community data required for whatever page we're on -- in the case of landing on a community, this should be only the active community. For e.g. the for you page, we may need more information.

The eventual place we want to land is converting app.config.chains and app.config.nodes to react query, which also means we must be able to query directly for chains/nodes, so I will make a ticket to split out queries for chains and nodes into separate routes. To begin with, we can query them in parallel alongside the remainder of the /status route, and eventually we can query them separately/as needed.
